observation/basic_health.py

```diff
@@ -106,7 +93,7 @@ class NoTargetsUpError(Exception):
                 orig_weights *= delay_weights  # unwrap the delays
                 amp_weights = np.abs(orig_weights)
                 phase_weights = orig_weights / amp_weights
-                new_weights[inp] = opts.fengine_gain * phase_weights.conj()
+                new_weights[inp] = fengine_gain * phase_weights.conj()
```

Comment:
does this work for multiple correlators?
Do they have the same solutions and requant gains?

Reply:
`session.set_fengine_gains` currently defaults to the 'wide' instrument only, so the narrowband instruments are left to their own devices. This should get us going without crashing things, but then we need to figure out what to do with the rest.